wurst/_handles/Force.wurst

```diff
@@ -42,3 +47,55 @@ public function force.containsUnit(unit whichUnit) returns boolean
 
 public function force.containsPlayer(player whichPlayer) returns boolean
 	return IsPlayerInForce(whichPlayer, this)
+
+int size
+public function force.size() returns int
+	size = 0
+	this.forEach()->
+		size++
+	return size
+
+class ForceIterator
+	private int offset
+	private int count
+	private int current
+
+	private static player array players
+	private static int index
+
+	construct(force f)
+		offset = this castTo int * bj_MAX_PLAYER_SLOTS
+		index = offset
+		f.forEach() ->
+			players[index] = GetEnumPlayer()
+			index++
+		count = index - offset
+		current = 0
+
+	function hasNext() returns bool
+		return current < count
+
+	function next() returns player
+		let p = current < count ? players[offset + current] : null
+		current++
+		return p
+
+	function close()
+		destroy this
+
+public function force.iterator() returns ForceIterator
+	return new ForceIterator(this)
+
+init
+	for i=0 to bj_MAX_PLAYER_SLOTS-1
```

Comment:
leave spaces between literals and operators, ie. `i = 0`

Same below for `()->` should be `() ->`.